client/src/AppBundle/Resources/views/Macros/macros-review.html.twig

```diff
@@ -191,9 +191,9 @@
 
             <table class="checkboxes labelvalue inline">
                 <tr>
-                    <td class="value checkbox">{% if param.answer == 'yes' %}X{% else %}&nbsp;{% endif %}</td>
+                    <td class="value checkbox"{% if param.answer == 'yes' %} aria-label=”Selected”>X{% else %}>&nbsp;{% endif %}</td>
```

Comment:
I'd check how this is read by AT as the aria-label on td can be problematic (see http://www.davidmacd.com/blog/does-aria-label-override-static-text.html). On a wider note, I'm not convinced by the table usage here, feels like it's solely for layout benefit.